electron/src/main.ts

```diff
@@ -235,8 +235,14 @@ const showMainWindow = async (mainWindowState: windowStateKeeper.State): Promise
     webappURL.hash = customProtocolHandler.hashLocation;
   }
 
-  if (argv.devtools) {
-    main.webContents.openDevTools({mode: 'detach'});
+  if (typeof argv.devtools !== 'undefined') {
+    if (typeof argv.devtools === 'number') {
```

Comment:
If the type is a number then it is also undefined, so we wouldn't need two if cases or am I missing something?